.coveralls.yml

```diff
@@ -0,0 +1 @@
+repo_token: fZagr53Gi12Qbd2F3H5i12bNZdnCWnFiD
```

Comment:
since this is a public repo, it is fine to do this, meanwhile we can remove this file, and add
COVERALLS_REPO_TOKEN to travisCI settings
Just as a precaution to remember not to commit tokens in repositories
I know the docs says  to add this file.

Reply:
TravisCI isn't setup yet. Thats why the build keeps failing.